tools/swift-plugin-server/Sources/CSwiftPluginServer/include/PluginServer.h

```diff
@@ -13,6 +13,9 @@
 #ifndef SWIFT_PLUGINSERVER_PLUGINSERVER_H
 #define SWIFT_PLUGINSERVER_PLUGINSERVER_H
 
+#include <stddef.h>
+#include <stdint.h>
+
```

Comment:
We need https://github.com/apple/swift/commit/4cb8133e4c0025473c09574a18f1a3f617fa0d6c as well I believe.